### ISSUE TITLE
vsUpdate triggered too often

### ISSUE THREAD

**Issue by whyboris** [rintoj/ngx-virtual-scroller#402]
Documentation for `vsUpdate` states:
> This event is fired every time the start or end indexes change

It seems to be triggered for every pixel change in the scroll position instead.

The issues is _not_ present for the `vsStart`, `vsEnd`, or `vsChange` events for example

**Comment by speige** [comment]
You're right, I think the documentation for vsChange & vsUpdate is backwards. Because vsChange states that it will run any time the scroll position changes.

Regardless, I'm not sure that it's necessary to call vsUpdate on every scroll position change. We should probably read through the history to see why we added that & see if it's really necessary.
